src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -52,31 +52,37 @@ public class TestJsonCommand : PSCmdlet
         private JsonSchema _jschema;
 
         /// <summary>
-        /// Process all exceptions in the AggregateException.
-        /// Unwrap TargetInvocationException if any and
-        /// rethrow inner exception without losing the stack trace.
+        /// Prepare a JSON schema.
         /// </summary>
-        /// <param name="e">AggregateException to be unwrapped.</param>
-        /// <returns>Return value is unreachable since we always rethrow.</returns>
-        private static bool UnwrapException(Exception e)
+        protected override void BeginProcessing()
         {
-            if (e.InnerException != null && e is TargetInvocationException)
-            {
-                ExceptionDispatchInfo.Capture(e.InnerException).Throw();
-            }
-            else
+            SchemaRegistry.Global.Fetch = static uri =>
```

Comment:
If it is static why not makes this class level static?

Also please add comment what the Fetch do and why we need it.

Reply:
> If it is static why not makes this class level static?

I'm not sure what you mean.  You [directed](https://github.com/PowerShell/PowerShell/pull/18023#discussion_r967962581) to make this static.

> Also please add comment what the Fetch do and why we need it.

The `Fetch()` function is [well-documented](https://json-everything.net/json-schema#automatic-resolution).  Do we really need a comment?

Reply:
> I'm not sure what you mean. You [directed](https://github.com/PowerShell/PowerShell/pull/18023#discussion_r967962581) to make this static.

I suggest to make it a static method so that we could reuse it in other cmdlets in future.

> The Fetch() function is [well-documented](https://json-everything.net/json-schema#automatic-resolution). Do we really need a comment?

It is very specific API and a comment could help a code reader. I suggest to take first paragraph from the doc.

Reply:
> I suggest to make it a static method so that we could reuse it in other cmdlets in future.

This method is very specific to JsonSchema.Net's validation operation.  I'm not sure it would useful outside of that context.